packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/simulate-utils.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+import { computed, Ref } from 'vue';
+import { WorkflowNode } from '@/types/workflow';
+import { DataArray } from '@/services/models/simulation-service';
+import { SimulateCiemssOperationState } from './simulate-ciemss-operation';
+
+export function usePreparedChartInputs(
```

Comment:
For simulate, how we save the async results are a bit different than other two operator so that it was easier to pass pyciemssMap as a props. In order to make it similar to other two operator, more refactoring is needed on the other parts as well. Not sure if it's worth doing it but we can revisit this later.